### ISSUE TITLE
ical support for calendar app import

### ISSUE THREAD

**Issue by houqp** [linkedin/oncall#1]

**Comment by Shuro** [comment]
What about the calendar function, will it be implemented? The two commits with the reference have nothing to do with this issue ticket.

**Comment by dwang159** [comment]
Yes, still planned. Not sure why those commits linked to this issue.
